dashboard/app/views/projects/projects.html.haml

```diff
@@ -15,8 +15,31 @@
       font-size: 1.5em;
     }
   }
-%div{'ng-controller' => 'ProjectsController'}
-  
+
+  .level-buttons {
+    margin: 20px 5px;
+  }
+
+%div
+  .alert
+    Hi! This is a skeleton page to see your projects. Visible to admins only ;)
+
+  %div
+    %span New project:
+    .btn-group.level-buttons{role: "group"}
+      %a.btn{href: ''}
+        (TODO) Standalone
+        %i.fa.fa-pencil
+      %a.btn{href: ''}
+        (TODO) Standalone
+        %i.fa.fa-rocket
+      %a.btn{href: '/s/course3/stage/21/puzzle/15'}
```

Comment:
Should this be `playlab_freeplay_path`?  https://github.com/code-dot-org/code-dot-org/blob/a7797b168fba6a10cf598da3c07979bf4038fb49/dashboard/app/views/layouts/_header.html.haml#L30